tequilapi/endpoints/service.go

```diff
@@ -116,14 +116,14 @@ func (se *ServiceEndpoint) ServiceList(resp http.ResponseWriter, _ *http.Request
 //       "$ref": "#/definitions/ErrorMessageDTO"
 func (se *ServiceEndpoint) ServiceGet(resp http.ResponseWriter, _ *http.Request, params httprouter.Params) {
 	id := service.ID(params.ByName("id"))
-	service := se.serviceManager.Service(id)
 
-	if service == nil {
-		utils.SendErrorMessage(resp, "Requested service not found", http.StatusNotFound)
+	serviceInstance := se.serviceManager.Service(id)
+	if serviceInstance == nil {
+		utils.SendErrorMessage(resp, "Requested serviceInstance not found", http.StatusNotFound)
```

Comment:
Resource is called `service`(`.ServiceGet()` method, `GET /services/:id` path), please use terms consistently at the API level. 
```suggestion
		utils.SendErrorMessage(resp, "Requested service not found", http.StatusNotFound)
```

Reply:
No point to align internal structures to exposed ones. API should not leak internal structures, what's why we map `service.Instance` -> `tequilapi.ServiceInfo`